File: drills/elixir/harnesses/nc02_valid_anagram.exs
[
  {~s|anagram?("anagram", "nagaram")|, inspect(true), inspect(Solution.anagram?("anagram", "nagaram"))},
  {~s|anagram?("rat", "car")|, inspect(false), inspect(Solution.anagram?("rat", "car"))},
  {~s|anagram?("", "")|, inspect(true), inspect(Solution.anagram?("", ""))},
  {~s|anagram?("a", "ab")|, inspect(false), inspect(Solution.anagram?("a", "ab"))}
]

File: drills/go/harnesses/nc02_valid_anagram.go
package main

func main() {
	run(func() []testCase {
		return []testCase{
			tc("isAnagram('anagram', 'nagaram')", true, isAnagram("anagram", "nagaram")),
			tc("isAnagram('rat', 'car')", false, isAnagram("rat", "car")),
			tc("isAnagram('', '')", true, isAnagram("", "")),
			tc("isAnagram('a', 'ab')", false, isAnagram("a", "ab")),
		}
	})
}
